build/automation/var/profile/local.mk

```diff
@@ -2,10 +2,22 @@
 
 # ==============================================================================
 # Service variables
-SPRING_PROFILES_ACTIVE := test,prod
-URL := "http://localhost:9095/dosapi/dosservices/v0.0.1/services/byfuzzysearch"
+SPRING_PROFILES_ACTIVE := local
```

Comment:
We shouldn't be creating Spring profiles as we use make profiles for that and drive everything from variables.